services/workflows-service/src/common/http-service/utils.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { AxiosError, isAxiosError } from 'axios';
+
+const _httpStatusFromAxiosMap = {
```

Comment:
Note the `_` prefix suggests this map is not in use.
```suggestion
const httpStatusFromAxiosMap = {
```

Reply:
is it in used here